codepropertygraph/src/main/scala/io/shiftleft/passes/CpgPass.scala

```diff
@@ -33,9 +31,17 @@ import scala.concurrent.duration.DurationLong
   *
   * @param cpg the source CPG this pass traverses
   */
-abstract class CpgPass(cpg: Cpg, outputName: String = getClass.getSimpleName) {
+abstract class CpgPass(cpg: Cpg, outName: String = "") {
   import CpgPass.logger
 
+  private val outputName = {
+    if (outName.isEmpty) {
+      this.getClass.getSimpleName()
+    } else {
+      outName
+    }
+  }
+
```

Comment:
Why's that better than before?

Reply:
Because before, it always returned `Predef$` for some reason...

Reply:
I mean that would make a good comment ...

Reply:
> I mean that would make a good comment ...

yes it would, just found myself scratching my head about this one :)